account-kit/infra/src/actions/types.ts

```diff
@@ -58,7 +58,9 @@ export type RequestGasAndPaymasterAndDataRequest = [
     policyId: string | string[];
     entryPoint: Address;
     dummySignature: Hex;
-    userOperation: UserOperationRequest;
+    userOperation: Omit<UserOperationRequest, "eip7702auth"> & {
+      authorizationContract?: Address;
```

Comment:
I forget, is this the final interface @andysim3d? I thought we were switching to the EF's `eip7702auth` everywhere.

Reply:
changing this now